docs/content/staking/rewards.mdx

```diff
@@ -4,10 +4,55 @@ sidebar_title: Rewards
 description: How the Flow protocol pays rewards to stakers
 ---
 
-# Rewards
+## Rewards
 
-Rewards are paid out at the end of each epoch to all the users that have tokens staked.
+Rewards are paid out at the end of each [epoch](/staking/#epochs) to all the users that have tokens staked.
 Rewards are considered to be unlocked and liquid, so the user can withdraw their rewards
 at any time.
 
-More information about the rewards calculation and schedule will be provided in the near future.
+If you staked using a 3rd party staking provider such as Kraken, Blocto or Finoa, please read [our guides](/staking/custody-providers/) for more information about how
+to claim and use your rewards via your chosen provider.
+
+## Rewards Distribution
+
+The rewards distribution schedule has been designed to ensure that enough liquid FLOW
+is available in the ecosystem to a wide variety of holders after the first few reward payments.
+Using this model, any user will be able to easily aquire FLOW from many different places
+in order to participate in the network.
+
+The numbers in this table represent the total tokens that are paid as rewards each epoch.
+They are not meant to indicate any expectations for an individual staker's rewards.
+Different variables go into calculating what an individual staker receives as rewards.
+
+|                          | Week 1         | Weeks 2-5  | Weeks 6 - 72 | Weeks 73+ |
+| ------------------------ | -------------- | ---------- | ------------ | --------- |
+| Total Rewards % (Annual) | 5%             | 20%        | 5%           | TBD      |
+| Total Rewards Amount     | 1,173,391 FLOW | ~4.4M FLOW | ~1.2M FLOW   | TBD
+
+## Using Rewards
+
+The process for collecting and using reward payouts will be different depending on how your tokens were staked.
+If you staked tokens using a custody provider, see [our guides](/staking/custody-providers/) for
+details about how to claim your rewards.
+
+Rewards will continue to accumulate in your account as long as you are staked and operate your node properly.
+You're not required to do anything with your rewards, but you can claim them and use them in a few ways.
+
+#### Once you've claimed your rewards you can:
+
+- Re-stake to earn compounding rewards! Most custody providers will automatically re-stake your rewards. (Some do not).
+  For details about re-staking related to your chosen custody provider see [our guides](/staking/custody-providers/).
+  **Note:** If you would like to stake a different node than you're currently staking, you may need to contact you custody provider directly.
```

Comment:
If you would like to stake a different node than the one you're currently staking, you may need to contact your custody provider directly